moddio/taro2
# Chore recompile box2dwasm and remove velocity limit

## Summary
- recompiled box2d.simd.wasm, only remove its velocity limit, u can see the change in https://github.com/moe-moe-pupil/box2d/commit/e0c9e8b3ef2ca25f137585663280b0f32570d9e8#diff-51ae01b561c6517d2dd23c662b65886cd5811b79a457ca9214d21b09a8de0887
- reset _scaleRatio to 60 (from 120) to avoid too small physics bodies
## how to test
I tested,
- two houses, use swordfish to push the bed, now it won't phase out the wall
[two_houses.txt](https://github.com/moddio/taro2/files/12439054/two_houses.txt)
- turn browser tab in inactive and turn it active back to see if the projectile will speed up, now it will
[item_bug_wasm.txt](https://github.com/moddio/taro2/files/12439061/item_bug_wasm.txt)
- Angry Assault, set _scaleRatio to 30 to see if there is any velocity limit, now it doesn't have any more
